libs/commons/Logs_.ml

```diff
@@ -304,7 +313,13 @@ let setup_logging ?(highlight_setting = Std_msg.get_highlight_setting ())
   Logs.Src.list ()
   |> List.iter (fun src ->
          match Logs.Src.name src with
-         | x when List.mem x skip_libs -> Logs.Src.set_level src None
+         | x
+           when List.exists
+                  (fun lib ->
+                    let re = Re.Pcre.regexp lib in
+                    Re.execp re x)
```

Comment:
Do we use `Re` elsewhere for this? We mix and match a lot of regex libraries so I would just want to confirm we don't use `Re` here and `Pcre` elsewhere on the same stuff.

Reply:
We already use Re.Pcre in this file, so was just being consistent. Can change it though

Reply:
Sounds fine then.